src/views/Admin/DogAdmin.js

```diff
@@ -0,0 +1,44 @@
+import { useState } from 'react';
+import { addDog } from '../../services/dogs';
+import { useHistory } from 'react-router-dom';
+import DogForm from '../../components/DogForm/DogForm';
+import './DogAdmin.css';
+
+export default function DogAdmin() {
+  const [name, setName] = useState('');
+  const [breed, setBreed] = useState('');
+  const [image, setImage] = useState('');
+  const [bio, setBio] = useState('');
+  const [message, setMessage] = useState('');
+
+  const history = useHistory();
+
+  const handleSubmit = async (e) => {
+    try {
+      e.preventDefault();
+      const data = await addDog(name, breed, image, bio);
+      setMessage('Success! Redirecting...');
+      setTimeout(() => {
```

Comment:
I might try to disable the submit button between while the page is re-directing to prevent a user from adding multiple dogs at once

Reply:
added!